__init__.py

```diff
@@ -622,12 +623,26 @@ def show_date_mark1(self, location, day):
     def get_weekday(self, day=None, location=None):
         if not day:
             day = self.get_local_datetime(location)
-        return day.strftime("%A")
+        weekday = nice_date(day)  # forces lingua_franca to initialize
+        if self.lang in date_time_format.lang_config.keys():
+             weekday = list(
+                 date_time_format.lang_config[self.lang]['weekday'].values())[day.weekday()]
```

Comment:
This is all correct but it's quite a mouthful, I wonder if we can split it to make it clearer for people reading the code in the future. Maybe something like:
```
localized_day_names = list(date_time_format.lang_config[self.lang]['weekday'].values())
weekday = localized_day_names[day.weekday()]
```

Reply:
In German weekday names are always capitalized, so again I assumed something which isn't always valid...